GenderPayGap.WebUI/Views/Submit/EmployerWebsite.cshtml

```diff
@@ -61,9 +61,10 @@
                 <p>You can publish your gender pay gap narrative on your organisation's website. This can provide context for your figures.</p>
                 <fieldset>
                     <div class="form-group @Html.SetErrorClass(model => model.CompanyLinkToGPGInfo, "error")">
-                        @Html.LabelFor(model => model.CompanyLinkToGPGInfo, "Enter the web address (URL)", new {@class = "form-label"})
+                        <label class="form-label" for="OtherName">Enter the web address (URL). <br/>

```

Comment:
`for="OtherName"` should probably be `for="CompanyLinkToGPGInfo"` - then, when you click on the label, it will highlight the input box

Reply:
Sorry, I should have spotted that. Fixed now